keyboards/rgbkb/sol3/rev1/config.h

```diff
@@ -9,6 +9,11 @@
 
 #pragma once
 
+#define SPLIT_WATCHDOG_ENABLE
+#define SPLIT_WATCHDOG_TIMEOUT 3000
+#define SPLIT_USB_TIMEOUT 2000
+#define SPLIT_USB_TIMEOUT_POLL 10
+
```

Comment:
`SPLIT_WATCHDOG_ENABLE` can go in `keyboard.json` as `split.transport.watchdog`, the rest duplicate the defaults
```suggestion
```

Reply:
@zvecr Thanks!

Reply:
`split.transport.watchdog` was never re-added?

Reply:
As this was resolved without a comment, we are to assume that its not needed?

If so, please could you update the PR title and description.

Reply:
No it is needed, the usb issues exist.

Reply:
Then you need to make the required changes to the `keyboard.json`.

Reply:
Tested locally, it works, although the delay between checks is questionable? Possibly needs an error noise.